blocks/stopwatch/package.json

```diff
@@ -15,17 +15,18 @@
     "fix:eslint": "eslint --fix .",
```

Comment:
This isn't on the hub at the moment, we probably don't want to publish it this time around either right?

Reply:
Not sure why it wasn't on – might as well publish it if it works